README.md

```diff
@@ -259,6 +271,41 @@ $ cordova run android
 
 At this point, the app named `Watson Vehicle Damage Analyzer` should be on your mobile device. Use the camera icon to take a photo of an automobile windshield, tire, vandalism, or of a motorcycle. The mobile application will send the image to the server after you click on the `check mark`, and the server will use Watson to analyze the image and fetch the results.
 
+## 8a. Add iOS platform and plugins
+
+Install the iOS deployment tools
+
+```
+    $ npm install -g ios-sim
+    $ npm install -g ios-deploy
+```
+
+Add the iOS platform and build. This will create an iOS folder in `platform` directory with all necessary files to run in emulator or iOS device
+
+```
+    $ cordova platform add ios
+    $ cordova prepare              # or "cordova build"
+```
+All cordova plugins are configured in [mobile/config.xml](mobile/config.xml) and will be installed when you create the platform and build.
+
+## 8b. Setup your iOS project
+
+In order to run the iOS project that was created from step #8a, we need to first create the `provisioning file,app IDs and certificates` from `Xcode`. You need to have an apple login which is free if you have an iOS device. Go to `Xcode>Preferences>Accounts` and add your apple login. This will create a `Personal Team` profile which can be used to sign your project.
+
+If you get `error: exportArchive: No profiles for ‘com.watson.vehicledamageanalyzer’ were found`. You need to select project in Xcode  and change the `bundle identifier` to a unique one. Also change the widget `id` in [mobile/config.xml](mobile/config.xml) to the same one in Xcode and unique
```

Comment:
"in Xcode and unique" -> "in Xcode."
I think the  "and unique" is extra?

Also, probably good to put an example here, i.e. say:
change `com.watson.vehicle-damage-analyzer` to your new bundle identifeier name `com.foo.vehicle-damage-analyzer`